client/src/explorer/gatewayExplorer.ts

```diff
@@ -177,9 +178,12 @@ export class BlockchainGatewayExplorerProvider implements BlockchainExplorerProv
                     arguments: [runtimeGateway]
                 };
 
+                let gatewayName: string;
+                gatewayName = FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME;
+
                 const treeItem: LocalGatewayTreeItem = await LocalGatewayTreeItem.newLocalGatewayTreeItem(
                     this,
-                    runtimeGateway.name,
+                    gatewayName,
```

Comment:
you can just put the `FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME` here you don't need to create a variable